main.js

```diff
@@ -1,163 +1,388 @@
+
+/*
+Opdracht 3 - Array methoden en functies
+Opdracht 3a: Gebruik een array-methode om alle tv merken (zoals Philips, NIKKEI, etc.)
+    in een lijst op de pagina weer te geven. Zorg ervoor dat dit ook zou werken als we 200 tv's in onze array zouden hebben staan.
+    Dat er dubbele namen in zitten, is niet erg.
+Opdracht 3b: Schrijf de code uit 3a om naar een functie die een array met tv-objecten verwacht.
+    Het is handig om onze scripts als functies op te zetten, zodat we ze gemakkelijk kunnen hergebruiken.
+    Tip: vergeet deze functie -declaratie niet aan te roepen!
+ */
+
+// Opdracht 3a:
+
+const tvBrands = inventory.map((tvBrand) => {
+  return `<li>${tvBrand.brand}</li>`;
+})
+const element4 = document.getElementById("listTvs");
+
+element4.innerHTML = tvBrands.join('');
+
+
+// Opdracht 3b:
+function tvObjects(inventory) {
+    const element4 = document.getElementById('listTvs'); // Koppeling HTML.
+
+    const tvBrands = inventory.map((tvBrand) => {
+        return `<li>${tvBrand.brand}</li>`; // Overschrijven HTML tekst in lijstvorm.
+    });
+
+    element4.innerHTML = `${tvBrands.join('')}`; // Maakt string van array zonder de komma's.
+}
+
+tvObjects(inventory); // Functie aanroepen.
+
+/*
+Opdracht 4 -Functies
+Maak deze gehele opdracht eerst alsof je het voor één tv doet. We gaan één tv weergeven in het volgende format:
+
+Philips 43PUS6504/12 - 4K TV
+€379,-
+43 inch (109 cm) | 50 inch (127 cm) | 58 inch (147 cm)
+
+Opdracht 4a: Maak een herbruikbare functie die een string genereert voor de naam van één tv en deze teruggeeft
+ in het format [merk] [type] - [naam] zoals Philips 43PUS6504/12 - 4K TV of NIKKEI NH3216SMART - HD smart TV.*/
+
+function infoTv(inventory) {
+    return `${inventory.brand} ${inventory.type} - ${inventory.name}`;
+}
+
+console.log(infoTv(inventory));
+
+
+//Opdracht 4b: Maak een herbruikbare functie die de prijs van één tv als parameter verwacht (zoals 379 of 159) teruggeeft in het format €379,- of €159,-.
+
+function priceTv(price) {
+    return `€${price},-`;
+}
+
+console.log(priceTv);
+
+/*
+Opdracht 4c: Maak een herbruikbare functie die een string genereert voor alle beschikbare schermgroottes van één tv.
+De functie geeft dit terug in het format [schermgrootte] inches ([schermgrootte omgerekend]cm) | [schermgrootte] inches
+([schermgrootte omgerekend]cm) etc. Als een tv maar één schermgrootte heeft ([32]) wordt de output 32 inch (81 cm).
+Wanneer een tv vier schermgroottes heeft ([43, 50, 55, 58]) wordt de output 43 inch (109 cm) | 50 inch (127 cm) | 58 inch (147 cm).
+Let op: om één string te genereren uit een array van schermgroottes zul je een for-loop voor moeten gebruiken.*/
+
+function screenSizes(inventory) {
+    let output = "";
+    for (let i = 0; i < inventory.length; i++) {
+        const currentSizeInches = inventory[i];
+        const currentSizeCm = inventory[i] * 2.54;
+
+        output = output + `${currentSizeInches} inch (${currentSizeCm} cm)`;
```

Comment:
Met string concatentation kan het ook
```suggestion
        output += `${currentSizeInches} inch (${currentSizeCm} cm)`;
```